test/test_dockerfile_based_harness.py

```diff
@@ -9,15 +9,17 @@
 _docker_client = docker.from_env()
 
 
-@pytest.mark.parametrize("dockerfile_path", ["keras.test.Dockerfile",
-                                             "matplotlib.test.Dockerfile",
-                                             "scipy.test.Dockerfile",
-                                             "numpy.test.Dockerfile",
-                                             "boto3.test.Dockerfile",
-                                             "pandas.test.Dockerfile",
-                                             "sm-python-sdk.test.Dockerfile",
-                                             "pytorch.examples.Dockerfile",
-                                             "tensorflow.examples.Dockerfile"])
+@pytest.mark.parametrize("dockerfile_path", [
+    "keras.test.Dockerfile",
+    "matplotlib.test.Dockerfile",
+    "scipy.test.Dockerfile",
+    "numpy.test.Dockerfile",
+    "boto3.test.Dockerfile",
+    "pandas.test.Dockerfile",
+    "pytorch.examples.Dockerfile",
+    "tensorflow.examples.Dockerfile",
+    "autogluon.test.Dockerfile",
```

Comment:
We should postpone inclusion of the new tests to the harness until autogluon itself is actually added to the SD runtime, otherwise we'll introduce avoidable failures to the test suite for any releases between now and the next minor version release.

Reply:
Shouldn't we merge in this PR after ag is added then?

Reply:
So this PR still holds value because of the actual test (which is dockerfile and the supplementary shell script). Justin is just asking you to not change the [test/test_dockerfile_based_harness.py](https://github.com/aws/sagemaker-distribution/pull/44/files/50e0f52e2dbeae31dc40e6e4cd903a10d94095cb#diff-df83b9248cc77eea573460ca513bcba05b318448d0e032cd9d09d69f88149958)

When we do the actual release, we will do two changes
* Modify env.in files to include AG
* Modify the above mentioned file to include AG tests (which will be dormant).

There is always a possibility that we might run into other errors while doing a minor version release. So if that happens (we are not able to generate a new minor version), then this change will block the ongoing patch releases.

Reply:
Makes sense. Just updated.